code/_globalvars/configuration.dm

```diff
@@ -36,3 +36,5 @@ GLOBAL_VAR_INIT(MAX_EX_FLAME_RANGE, 14)
 GLOBAL_PROTECT(MAX_EX_FLAME_RANGE)
 GLOBAL_VAR_INIT(DYN_EX_SCALE, 0.5)
 
+//Screen resolution stuffs.
+GLOBAL_VAR_INIT(DEFAULT_VIEW, "15x15")

```

Comment:
You are just mirroring the config though, just use CONFIG_GET(string/default_view) instead of GLOB.DEFAULT_VIEW